html/status.html

```diff
@@ -358,7 +358,7 @@
 			}
 		}
 
-		function getTable(state) {
+		function getTable(state,oldstate) {
```

Comment:
Can you please add spaces between parameters (same thing where this function is called further down)?

Reply:
Fixed! So please resolve